web/packages/teleport/src/Main/MainContainer.tsx

```diff
@@ -29,11 +29,9 @@ export const MainContainer = styled.div`
   --sidebar-width: 256px;
   --sidenav-width: 76px;
   --sidenav-panel-width: 224px;
+  overflow: hidden;
```

Comment:
this seems like a pretty big property to change on the `MainContainer`. But I'm sure we'll catch anything crazy in the test plan